pkg/matcher/annotation_matcher.go

```diff
@@ -71,13 +71,15 @@ func getAnnotationValues(annotation string) ([]string, error) {
 
 	// if it's not an array then it would be a single string
 	if !strings.HasPrefix(annotation, "[") {
-		return []string{annotation}, nil
+		// replace &#44; with comma so users can have comma in the annotation
+		annot := strings.ReplaceAll(annotation, "&#44;", ",")
```

Comment:
Will it replace when `,` is inbetween name such as [main, release,nightly]
OR
does its replace in this scenario as well  [main, release]

Reply:
strings.Split doesn't consider spaces... so [main, release,nightly] means any of those branches.